client/src/components/ControlPanel.vue

```diff
@@ -511,6 +484,19 @@ export default {
                         </v-slider>
                       </v-col>
                     </v-row>
+                    <v-row>
+                      <v-col cols="4">
+                        Display crosshairs
+                      </v-col>
+                      <v-col cols="8">
+                        <v-checkbox
```

Comment:
I'm a fan of `v-switch` because it looks sophisticated, but nbd

Reply:
Yeah, that works too! I agree that it looks better.